sys/mips/include/pte.h

```diff
@@ -242,6 +246,9 @@ typedef uint64_t pd_entry_t;
 #define	PTE_PS_16M		((pt_entry_t)0x30 << TLBLO_SWBITS_SHIFT)
 #define	PTE_PS_64M		((pt_entry_t)0x38 << TLBLO_SWBITS_SHIFT)
 #define	PTE_PS_IDX_MASK		((pt_entry_t)0x38 << TLBLO_SWBITS_SHIFT)
+#if defined(CPU_CHERI)
```

Comment:
Hmph are we still using that in sys/mips? :(

Reply:
Looks like `sys/mips` is a mix of `defined(CPU_CHERI)` and `__has_feature(capabilities)`.  In order to not make the mess worse I can use the latter if that's preferable?

Reply:
What's there is inconsistent, AFAICT.  Some `ifdef CPU_CHERI`, some `has_feature(capabilities)`, sadly.  `PTE_SCI` is currently guarded by `ifdef CPU_CHERI` and so I just went from there.